src/components/StyleGuide/Colors.stories.mdx

```diff
@@ -6,82 +6,387 @@ import SimpleGrid from "../Grid/SimpleGrid";
 import { getCategory } from "../../utils/componentCategories";
 import DSProvider from "../../theme/provider";
 
+export const colorContrastData = {
+  brand: {
+    primary: {
+      dataBlack: ["3.45", "Fail", "AA"],
```

Comment:
Just wondering – is there a formula to derive this information, or is hardcoding it the best option?